tests/helpers/reference_types_helpers.test.ts

```diff
@@ -68,4 +71,31 @@ describe("loopThroughReferenceType", () => {
     );
     expect(loopThroughReferenceType(refToken("Sheet2!$A1:$B1"))).toEqual(refToken("Sheet2!A1:B1"));
   });
+
+  describe("setXcToFixedReferenceType", () => {
+    test.each(["A1", "$A1", "A$1", "$A$1"])("simple ref", (ref) => {
+      expect(setXcToFixedReferenceType(ref, "none")).toBe("A1");
+      expect(setXcToFixedReferenceType(ref, "col")).toBe("$A1");
+      expect(setXcToFixedReferenceType(ref, "row")).toBe("A$1");
+      expect(setXcToFixedReferenceType(ref, "colrow")).toBe("$A$1");
+    });
+
+    test.each(["A1", "$A1", "A$1", "$A$1"].map((tl) => `Sheet!${tl}`))("with sheetName", (ref) => {
```

Comment:
much much easier to see what the test is testing
```suggestion
    test.each(["Sheet!A1", "Sheet!$A1", "Sheet!A$1", "Sheet!$A$1"])("with sheetName", (ref) => {
```